recipe/meta.yaml

```diff
@@ -30,15 +31,17 @@ requirements:
     - absl-py >=0.1.6
     - astor >=0.6.0
     - gast >=0.2.0
-    - numpy >=1.13.3
+    - numpy >=1.16.1
```

Comment:
The wheel indicates compatibility with nuimpy >=1.13.3

Reply:
I was unable to get the import to work due to an issue similar to https://github.com/tensorflow/tensorflow/issues/559, which was resolved by a numpy upgrade.